spyder/plugins/application/container.py

```diff
@@ -302,31 +302,40 @@ def _check_updates_ready(self):
                 header = _("<b>Spyder {} is available!</b> "
                            "<i>(you have {})</i><br><br>").format(
                     latest_release, __version__)
-                content = ""
+                download_msg = _(
+                    "Would you like to automatically download and "
+                    "install it?<br><br>"
+                )
+                pip_warn_msg = _(
+                    "<b>Important note:</b> Since you installed "
+                    "Spyder with Anaconda,{} please <b>don't</b> use "
+                    "<code>pip</code> to update it as that will break "
+                    "your installation.<br><br>"
+                    "Instead, run the following commands in a "
+                    "terminal:<br>"
```

Comment:
```suggestion
                    "Spyder with Anaconda, please <b>don't</b> use "
                    "<code>pip</code> to update it as that will break "
                    "your installation.<br><br>"
                    "Instead,{} run the following commands in a "
                    "terminal/Anaconda Prompt:<br>"
```

* Move the inject to a more appropriate place
* Add "Anaconda Prompt" to cover Windows

Reply:
I agree with the anaconda prompt, but disagree with the moving the first injection to a later position. If anything the first injection should be moved further toward the beginning: the qualifier "if not downloading" should be one of the first things the user reads before reading further.